subproc.cc

```diff
@@ -322,6 +322,7 @@ static int reapProc(nsjconf_t* nsjconf, pid_t pid, bool should_wait = false) {
 			LOG_I("pid=%d (%s) terminated with signal: %s (%d), (PIDs left: %d)", pid,
 			    remote_txt.c_str(), util::sigName(WTERMSIG(status)).c_str(),
 			    WTERMSIG(status), countProc(nsjconf) - 1);
+			fprintf(stderr, "Program terminated with signal: %s", util::sigName(WTERMSIG(status)).c_str());
```

Comment:
This is great but could a user program can print this message too and confuse things? Should it have a newline?

```suggestion
			fprintf(stderr, "Program terminated with signal: %s\n", util::sigName(WTERMSIG(status)).c_str());
```

Reply:
Maybe we should use the nsjail fatal log method so it's more easily identifiable? (Or just up the existing log message to fatal?)
I'm pretty sure we keep the fatal level logs.